azure-pipelines.nuget.yml

```diff
@@ -1,213 +1,213 @@
 trigger: none
 
 parameters:
-  - name: version
-    displayName: Version to stamp on binaries and nuget package. Should be in form of "major.minor", for example "1.0"
-    type: string
-
-pool:
-  vmImage: "windows-latest"
+- name: version
+  displayName: Version to stamp on binaries and nuget package. Should be in form of "major.minor", for example "1.0"
+  type: string

```

Comment:
Why are you removing the pool? I think you need to sync your branch with the latest changes from master